### ISSUE TITLE
Default OAUTH_EXPIRY_SECONDS value is incorrect

### ISSUE THREAD

**Issue by justmobilize** [freakboy3742/pyxero#230]
[Public applications](https://developer.xero.com/documentation/auth-and-limits/public-applications) have a default expiration of 30 minutes (`1800`), not 60 minutes (`3600`)

Is there a reason for the higher value?

**Comment by Wozzyy** [comment]
@freakboy3742 I don't think this is relevant any more - public apps are part of the old OAuth 1.0 flow.

Suggest closing unless I'm mistaken.

**Comment by freakboy3742** [comment]
Seems reasonable to me - thanks for the housekeeping!
